diff --git a/dist/module.js b/dist/module.js
index b3e3992e52cb0f7c533ccef8ba3411d2e474b34b..45fd4651e801b0293ec9e85437d6a59271d165d0 100644
--- a/dist/module.js
+++ b/dist/module.js
@@ -51,7 +51,8 @@ class $279c20a4a0c8d128$export$cacbb3924155d68e extends $dqiJs$Layout {
     buildCollection() {
         let y = this.padding;
         let nodes = [];
-        for (let node of this.collection){
+        const visibleNodes = [...this.collection.getKeys()].map(key => this.collection.getItem(key));
+        for (let node of visibleNodes){
             let layoutNode = this.buildChild(node, 0, y);
             y = layoutNode.layoutInfo.rect.maxY;
             nodes.push(layoutNode);
