diff --git a/node_modules/@react-aria/overlays/dist/module.js b/node_modules/@react-aria/overlays/dist/module.js
index 1ef0a15..66b81bb 100644
--- a/node_modules/@react-aria/overlays/dist/module.js
+++ b/node_modules/@react-aria/overlays/dist/module.js
@@ -311,7 +311,7 @@ function $df1b2f080f58bf9e74aeb666b8c96ba$export$useCloseOnScroll(opts) {
       // Ignore if scrolling an scrollable region outside the trigger's tree.
       let target = e.target;
 
-      if (!triggerRef.current || !target.contains(triggerRef.current)) {
+      if (!triggerRef.current || !(e.target instanceof HTMLElement) || !target.contains(triggerRef.current)) {
         return;
       }
 
