diff --git a/node_modules/@react-aria/overlays/dist/module.js b/node_modules/@react-aria/overlays/dist/module.js
index a7d107b..b4f2ec5 100644
--- a/node_modules/@react-aria/overlays/dist/module.js
+++ b/node_modules/@react-aria/overlays/dist/module.js
@@ -232,7 +232,7 @@ function $57cf16c65f4d0c5c$export$18fc8428861184da(opts) {
             // Ignore if scrolling an scrollable region outside the trigger's tree.
             let target = e.target;
             // window is not a Node and doesn't have contain, but window contains everything
-            if (!triggerRef.current || target instanceof Node && !target.contains(triggerRef.current)) return;
+            if (!triggerRef.current || !(target instanceof Node) || !target.contains(triggerRef.current)) return;
             let onCloseHandler = onClose || $57cf16c65f4d0c5c$export$f6211563215e3b37.get(triggerRef.current);
             if (onCloseHandler) onCloseHandler();
         };
